test/dynamo/test_replay_record.py

```diff
@@ -32,14 +31,14 @@ def setUpClass(cls):
         cls._exit_stack.enter_context(
             unittest.mock.patch.object(
                 torch._dynamo.config,
-                "replay_record_dir_name",
-                "/tmp/torch._dynamo_error_records/",
+                "debug_dir_root",
+                "/tmp/_torchdynamo_debug_/",
```

Comment:
`_torchdynamo_debug_' + os.environ.get( ” USERNAME” )`?

Reply:
This is just test code which gets cleaned up afterward, so I'll leave this if you're okay with it.

Reply:
sounds good

Reply:
I'd prefer to use ```+ USERNAME``` as many users working on share cluster like us, other folks may pollute or overwrite your debug or tracing output.